examples/next/docs/react/basic-example/src/index.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from "react-dom/client";
 import './index.css';
 import App from './App';
 
-ReactDOM.render(
-  <App />,
-  document.getElementById('root')
-);
+const container = document.getElementById("container");
```

Comment:
According to 1 and 2 points. I don't know why it hasn't been reverted 🙄 Changed within https://github.com/handsontable/handsontable/pull/10300/commits/0a3d06e9d72930c7dc15e8af183bffc6d2a6f678.

3. I thought that problem is only related to our linking system and examples, but it seems that there are some problems with ES build. Thus, I can agree that it should be fixed sooner than later. Probably it will be a breaking change.